DigitalLearningSolutions.Data.Tests/Services/CourseServiceTests.cs

```diff
@@ -417,37 +417,63 @@ public void
         }
 
         [Test]
-        public void RemoveDelegateFromCourse_removes_delegate_from_course()
+        public void DelegateHasCurrentProgress_returns_true_if_delegate_has_current_progress()
         {
             // Given
             A.CallTo(() => progressDataService.GetDelegateProgressForCourse(1, 1)).Returns(
                 new List<Progress> { new Progress { ProgressId = 1, Completed = null, RemovedDate = null } }
             );
 
             // When
-            var result =
-                courseService.RemoveDelegateFromCourseIfDelegateHasCurrentProgress(1, 1, RemovalMethod.RemovedByAdmin);
+            var result = courseService.DelegateHasCurrentProgress(1, 1);
 
             // then
             result.Should().BeTrue();
+        }
+
+        [Test]
+        public void DelegateHasCurrentProgress_returns_false_if_delegate_has_no_current_progress()

```

Comment:
Could we have another unit test for the case where GetDelegateProgressForCourse returns progress records, but only ones that are completed/removed.